lib/encoder-stream.js

```diff
@@ -19,6 +20,8 @@ class EncoderStream extends Transform {
 	}
 
 	_transform(object, encoding, callback) {
+		object.tags.AWS_KINESISVIDEO_PRODUCER_TIMESTAMP += (this.count * (1000 / 25)).toString();
```

Comment:
25 should not been hardcoded here, it should be given as input somewhere.

Some stream will be 25fps but others may be 30 fps or 12 fps, there is no general rule and the kvs-parser should be generic

Reply:
Yes @piercus I have tested this with the unit test it is producing a different timestamp every time

> 25 should not been hardcoded here, it should be given as input somewhere.
Yes we can take it from the FrameStream input

Reply:
> Yes @piercus I have tested this with the unit test it is producing a different timestamp every time

Do you understand why ? if not please create a specific issue, and a branch with a unit test which make this failing